.github/workflows/update-package-lock.yaml

```diff
@@ -0,0 +1,27 @@
+name: Update package-lock.json
+
+on:
+    schedule:
+        # This is probably 6am UTC, which is 10pm PST or 11pm PDT
+        # Alternatively, 6am local is also fine
+        - cron: '0 6 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 12
+        registry-url: https://registry.npmjs.org/
+
+    - name: Configure git and update package-lock.json
+      run: |
+        git config user.email "ts_bot@rcavanaugh.com"

```

Comment:
@RyanCavanaugh This is the right email address for the bot?

Reply:
typescriptbot@microsoft.com (see https://github.com/typescript-bot)

Reply:
I'll go ahead and update the other workflows?

Reply:
Done.  Shout if that's a problem.

Reply:
Its avatar is associated with that address on https://en.gravatar.com/ right? That's really the only visible part.

Reply:
@weswigham which address is on gravatar?

Reply:
The email associated with the commit. GitHub uses gravatar to find an avatar for it to display on the GitHub UI.